Task3Math2/Task3Math2/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task3Math2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Условие Задачи 21: Дано четырехзначное число (ввод с клавиатуры). Найти число, полученное из исходного путем вычеркивания третьей цифры и приписывания ее в начало числа.\r\n");
+            Console.WriteLine("Введите четырёхзначное число: \r\n");
+
+            int Main_number = int.Parse(Console.ReadLine());
+
+            // Для выполнения задачи я разобью число на разряды стандартным способом, а потом соберу новое число в консоли. 
+
+            int last_digit = Main_number % 10;
+
+            Main_number = Main_number / 10;
+
+            int third_digit = Main_number % 10;
+
+            Main_number = Main_number / 10;
```

Comment:
Не используйте snake_case в С# (такое именование в C# в весьма специфических случаях). Используйте camelCase. Также лучше не портить исходные данные, для изменяемой величины завести новую переменную. Обратите внимание на синтаксис деления с присваиванием. Кроме того, в этом участке кода пустые строки не нужны. Они только затрудняют чтение. Пустыми строками хорошо выделять относительно крупные смысловые участки кода. Это как автор разбивает текст литературного произведения на абзацы. Представьте, как было бы ужасно читать, если бы каждое предложение было бы в отдельном абзаце.
```suggestion
            int temp = mainNumber
            int lastDigit = temp % 10;
            temp /= 10;
            int third_digit = temp % 10;
            temp /= 10;
```